setup.py

```diff
@@ -22,7 +22,7 @@
 
 setup(
     name="roiextractors",
-    version="0.5.7",
+    version="0.5.6",
```

Comment:
Mmmm shouldn't it be `0.5.7` though? There's also no harm at all in jumping one as a test to `0.5.8`

Reply:
Ok, I was going to trigger the workflow one more time (0.5.6-->0.5.7) just to be extra sure it works, but I guess it's even better just to cut a new release and check that everything goes automatically